pkg/networkmanager/index.html

```diff
@@ -80,6 +80,7 @@
         <span translatable="yes">Interfaces</span>
         <div class="pull-right">
           <button translatable="yes" id="networking-add-bond" class="btn btn-default network-privileged">Add Bond</button>
+          <button translatable="yes" id="networking-add-team" class="btn btn-default network-privileged">Add Team</button>
```

Comment:
> How should people decide between these two buttons? Is it clear to users?

I think it is not clear.  This is the minimal level that allows us to claim the promised "team support" in Cockpit, and it is slightly better than what nmtui and Gnome Control Center have.